mhctools/mhcflurry.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2017. Mount Sinai School of Medicine
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function, division, absolute_import
+import logging
+
+from .base_predictor import BasePredictor
+from .binding_prediction import BindingPrediction
+from .binding_prediction_collection import BindingPredictionCollection
+
+from mhcflurry import Class1AffinityPredictor
+from mhcflurry.encodable_sequences import EncodableSequences
+
+logger = logging.getLogger(__name__)
+
+
+class MHCFlurry(BasePredictor):
+    """
+    Wrapper around MHCFlurry. Assumes you've installed it and downloaded datasets and trained  MHCFlurry: see
```

Comment:
If 'mhcflurry' is added as a requirement to mhctools I don't think you need to say "assumes you've instealled it". Also in any case no need to say " and trained  MHCFlurry": if they've downloaded datasets they don't need to train anything.